lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -341,7 +345,13 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
     ~H"""
     <div class="flex gap-4 sm:gap-12 border border-gray-200 rounded-lg p-4">
       <.labeled_text label="Slug">
-        /<%= @app_deployment.slug %>
+        <.link
+          href={@deployment_group.url <> ~p"/apps/#{@app_deployment.slug}"}
```

Comment:
Worth doing?

```suggestion
          href={@deployment_group.url |> URI.parse() |> URI.append_path(~p"/apps/#{@app_deployment.slug}")}
```

Reply:
We guarantee it doesn't end with a trailing slash, so we are good to go as is!